source/comparators/BySeriesAndIndex.h

```diff
@@ -53,8 +53,8 @@ class BySeriesAndIndex<Ship> {
 template<>
 class BySeriesAndIndex<Outfit> {
 public:
-	bool operator()(const std::string &nameA, const std::string &nameB)
-	{
+	bool operator()(const std::string &nameA, const std::string &nameB) const
+	 {
```

Comment:
```suggestion
	{
```
I don't think this whitespace is needed?

Reply:
oops

Reply:
no problem